src/puck.ts

```diff
@@ -413,7 +427,11 @@ export class LookupPuck {
   };
 
   private readonly onPuckSingleClick = () => {
-    // TODO: toggle whether the puck is enabled or disabled.
+    this.setEnabledState(
+      this.enabledState < LookupPuckEnabledState.enableGesturesAndLookup
+        ? LookupPuckEnabledState.enableGesturesAndLookup
+        : LookupPuckEnabledState.enableGestures
+    );
```

Comment:
Also in two minds about whether a more verbose if-else-if would be preferred as more readable.

And again, `<` versus `===` comparators.